### DISCUSSION TITLE
All chromium and electron based passwords / authentication tokens are wiped weirdly after installing explorer patcher

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#3577] [Q&A]
I'm not too sure if its **technically** explorer patcher but it is a bit weird that all of my saved passwords in Brave and all of my auth tokens are wiped on Discord ironically after using explorer patcher and not installing any other program.

The only problem I have with my computer is it blue screening daily because of faulty drivers (DPC_WATCHDOG_VIOLATION)
I really don't want to find out that smartscreen was correct and ep is malware.. I would hate to get a fork of StealC on my system from using the windows 10 start menu :/

![image](https://github.com/user-attachments/assets/9b779b54-da1c-40c5-bb64-4b474d741f0b)
(This screenshot is taken after I had tried to recover what I could)

**Comment by pyrates999** [comment]
Where did you download EP from?  Was it from here? https://github.com/valinet/ExplorerPatcher/releases

**Reply by ghost** [reply]
yep thats where I got it, i'm just really worried cuz im thinking of moving to a password manager (bitwarden) now and I have to change all my passwords that i.. lost- ;-;
im not tryna like say EP is a virus im just stressed out sorry

**Comment by pyrates999** [comment]
What version of windows are you running?  Please include the build number.  You can find out by running winver.

**Reply by ghost** [reply]
Windows 11 Home Build 22621.ni_release.220506-1250
or 23H2 22631.4037
![image](https://github.com/user-attachments/assets/d4a49e83-851a-4865-89c9-15554b435b65)
![image](https://github.com/user-attachments/assets/37aa7419-579f-47ad-a333-eb291305134c)

**Comment by pyrates999** [comment]
Check this: https://community.brave.com/t/my-autofill-passwords-have-disappeared-how-do-i-get-them-back/433367/12

If your computer is blue screening daily, you might have hardware issues then.

**Reply by ghost** [reply]
i've since moved to firefox and replaced the majority of the passwords, thanks for the support
i think the issue with my computer is drivers?
